packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -19,10 +19,11 @@
       data-testid="files-collaborators-no-share-permissions-message"
       v-text="noSharePermsMessage"
     />
-    <div v-if="hasSharees" class="avatars-wrapper oc-flex oc-flex-middle oc-flex-between">
-      <h4 class="oc-text-bold oc-my-rm" v-text="sharedWithLabel" />
-    </div>
     <template v-if="hasSharees">
+      <div id="files-collaborators-headline" class="oc-flex oc-flex-middle oc-flex-between">
+        <h4 class="oc-text-bold oc-my-rm" v-text="sharedWithLabel" />
+        <CopyPrivateLink :resource="resource" />
```

Comment:
There are a 2 more occurrences in this file and in `SpaceMembers.vue`.

```suggestion
        <copy-private-link :resource="resource" />
```